topics/command-info.md

```diff
@@ -0,0 +1,96 @@
+# Command information
+
+Starting from Redis 7.0 we include an additional map in `COMMAND`'s
+reply, which contains the following fields:
+ - `summary`
+ - `since`
+ - `group`
+ - `complexity`
+ - `doc-flags`
+ - `deprecated-since`
+ - `replaced-by`
+ - `history`
+ - `hints`
+ - `arguments`
+ - `key-specs`
+ - `subcommands`
```

Comment:
```suggestion
Starting from Redis 7.0 we include an additional map in `COMMAND`'s reply, which contains the following fields:

 - `summary`
 - `since`
 - `group`
 - `complexity`
 - `doc-flags`
 - `deprecated-since`
 - `replaced-by`
 - `history`
 - `hints`
 - `arguments`
 - `key-specs`
 - `subcommands`
```